src/pages/task/taskTabs/testsTable/TestsTableCore.tsx

```diff
@@ -44,6 +44,14 @@ export const TestsTableCore: React.FC = () => {
   const queryVariables = getQueryVariables(search, resourceId);
   const { cat, dir, pageNum, limitNum } = queryVariables;
 
+  useEffect(() => {
+    if (cat === undefined) {
+      updateQueryParams({
+        [RequiredQueryParams.Category]: TestSortCategory.Status,
+        [RequiredQueryParams.Sort]: SortDirection.Asc,
+      });
+    }
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I think it's okay to add cat to the dep array since it should never be omitted

Reply:
I purposefully omitted it so that we only default sort status if its the first time the component renders. We dont want to force the user to sort by status if they want to remove all sorts intentionally.